templates/download/shared/_server_weekly_news.html

```diff
@@ -0,0 +1,68 @@
+      </div>
+    </div>
+
+    <div class="col-4 col-start-large-9">
+      <!-- MARKETO FORM -->
+      <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_1917" class="marketo-form">
+        <fieldset style="border: 0; margin: 0; padding: 0;">
+          <ul class="p-list">
+            <li class="mktFormReq mktField p-list__item">
+              <label for="firstName" class="mktoLabel">First name:</label>
+              <input required  id="firstName" name="FirstName" maxlength="255" type="text" class="mktoField mktoRequired" >
+            </li>
+            <li class="mktFormReq mktField p-list__item">
+              <label for="lastName" class="mktoLabel">Last name:</label>
+              <input required  id="lastName" name="LastName" maxlength="255" type="text" class="mktoField mktoRequired">
+            </li>
+            <li class="mktFormReq mktField p-list__item">
+              <label for="company" class="mktoLabel">Company Name:</label>
+              <input required  id="company" name="Company" maxlength="255" type="text" class="mktoField mktoRequired">
+            </li>
+            <li class="mktFormReq mktField p-list__item">
+              <label for="email" class="mktoLabel ">Work email:</label>
+              <input required  id="email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField mktoRequired">
+            </li>
+            <li class="mktField p-list__item">
+              <input class="mktoField" value="yes" id="CanonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
+              <label class="mktoLabel mktoHasWidth" for="CanonicalUpdatesOptIn"><small>I would like to receive occasional news from Canonical by email.</small></label>
+            </li>
+            <li><small>All information provided will be handled in accordance with the Canonical <a href="/legal">privacy policy</a>.</small></li>
+            <li class="p-list__item">
+              <div class="g-recaptcha" data-sitekey="6LfYBloUAAAAAINm0KzbEv6TP0boLsTEzpdrB8if" style="margin: 2rem 0;"></div>
+            </li>
+            <li class="mktField p-list__item">
+              <button type="submit" class="mktoButton p-button--positive">Subscribe and download</button>
+              <input type="hidden" name="formid" class="mktoField" value="1917">
+              <input type="hidden" name="formVid" class="mktoField" value="1917">
+              <input type="hidden" name="munchkinId" class="mktoField" value="066-EOV-335">
+              <input type="hidden" name="Consent_to_Processing__c" value="yes" />
+              <input type="hidden" name="download_asset_url" value="https://assets.ubuntu.com/v1/fe2cb442-IoTSecurityWhitepaper-FinalReport.zip" aria-label="">
+              <input type="hidden" name="thankyoumessage" value="Thank you<br />Enjoy the cheat sheet!" aria-lable="">
```

Comment:
As far as I know, there is no effect for:
- `aria-label=""` for something that doesn’t already have a label to override
- _any_ `aria-label` for `<input type="hidden">`
- `aria-lable`, since it’s a misspelling.

More importantly: The resulting notification begins “Success: Thank you”, which is bizarre. (I guess “Success:” is cargo-culted from [the Vanilla example](https://github.com/canonical-web-and-design/vanilla-framework/blob/master/docs/examples/patterns/notifications/positive.html), so I’ll submit a fix for that shortly.) How about “Thanks for subscribing.”?

Reply:
> I guess “Success:” is cargo-culted from the Vanilla example, so I’ll submit a fix for that shortly

Submitted [vanilla-framework#2664](https://github.com/canonical-web-and-design/vanilla-framework/pull/2664).

Reply:
Nice, done.